packages/openapi-ts/test/__snapshots__/test/generated/v3_client_transform/services.gen.ts.snap

```diff
@@ -0,0 +1,56 @@
+// This file is auto-generated by @hey-api/openapi-ts
+
+import type { CancelablePromise } from './core/CancelablePromise';
+import type { BaseHttpRequest } from './core/BaseHttpRequest';
+import { ModelWithDatesResponse, ModelWithDatesArrayResponse, type ArrayOfDatesResponse, type DateResponse } from './types.gen';
+
+export class DefaultService {
+    constructor(public readonly httpRequest: BaseHttpRequest) { }
+    
+    /**
+     * @returns ModelWithDates Success
+     * @throws ApiError
+     */
+    public modelWithDates(): CancelablePromise<ModelWithDatesResponse> {
+        return this.httpRequest.request({
+            method: 'PUT',
+            url: '/api/model-with-dates',
+            responseTransformer: ModelWithDatesResponse
+        });
+    }
+    
+    /**
+     * @returns ModelWithDates Success
+     * @throws ApiError
+     */
+    public modelWithDatesArray(): CancelablePromise<ModelWithDatesArrayResponse> {
+        return this.httpRequest.request({
+            method: 'PUT',
+            url: '/api/model-with-dates-array',
+            responseTransformer: ModelWithDatesArrayResponse
+        });
+    }
+    
+    /**
+     * @returns string Success
+     * @throws ApiError
+     */
+    public arrayOfDates(): CancelablePromise<ArrayOfDatesResponse> {
```

Comment:
Any inline types (date[] in this case) right now are being missed out because they don't receive a Model due to not having a openapi Schema type. In my experience it's rare in OpenAPI to not have schemas emitted for a response type, but do we accept this as a limitation of the first version just to get this out there? 

It's probably solvable (maybe we generate synthetic schemas for inline types?) just not as much of a natural fit to how this library works right now

Reply:
Can you give an example of such schema?

Reply:
v3-transforms.json has the source for these generations in, it's a legal openapi schema, but I suppose more likely to be a hand-crafted one than come from c#/java etc. Probably not a huge priority to cover from day 1

Reply:
So I suggest we leave this for now, once it's merged then added support for additional corner cases will be easy to iterate toward